examples/graphql-interfaces/graphql_interfaces.client.out

```diff
@@ -1,5 +1,5 @@
 # Send a query to the GraphQL endpoint using a cURL command.
-# The query used: { characters { name } }
-$ curl -X POST -H "Content-type: application/json" -d '{ "query": "{ characters { name } }" }'
+# The query used: { animals { name } }
+$ curl -X POST -H "Content-type: application/json" -d '{ "query": "{ animals { name } }" }'
 'http://localhost:4000/graphql'
```

Comment:
The service path is not matched with the service.

Reply:
Updated